runtime/src/main/java/org/corfudb/runtime/collections/TxBuilder.java

```diff
@@ -30,6 +32,7 @@
     private final String namespace;
     private Timestamp timestamp;
     private final List<Runnable> operations;
+    private static final int DEFAULT_NUM_TX_RETRIES = 3;
```

Comment:
can you please make it not static? When it's not static we can make this parameter configurable, whet it's static it's hard coded

Reply:
the parameter is already configurable at runtime via an explicit argument right?
This is an internal value if the explicit argument is not set.